prime-router/src/main/kotlin/cli/tests/SftpcheckTest.kt

```diff
@@ -0,0 +1,155 @@
+ *  run after 'ping' and before 'end2end'.
+ */
+
+private const val jsonMimeType = "application/json"
+
+class SftpcheckTest : CoolTest() {
+    override val name = "sftpcheck"
+    override val description = "Test SFTP receiver connections"
+    override val status = TestStatus.DRAFT
+
+    /**
+     * Define private local variables for use in the test.
+     */
+    private val sftpcheckUri = "/api/check?sftpcheck="
+    private val ignoreReceiverNamesURI = "/api/settings/organizations/ignore/receivers"
+    private val sftpcheckMessage = "Test SFTP receiver connections: "
+    var sftpcheckTestResult = true
+
+    override suspend fun run(environment: Environment, options: CoolTestOptions): Boolean {
+        ugly("Starting SFTPCHECK Receciver Connections test ${environment.url}")
+
+        /**
+         * Get receiver ignore organizations with transport.host=sftp
+         */
+        val accessToken = getAccessToken(environment) // Get accessToken per environment.
+        val ignoreReceiverNamePath = environment.formUrl(ignoreReceiverNamesURI).toString()
+        val ignoreReceiversNameList = listReceiverNames(ignoreReceiverNamePath, accessToken)
+
+        /**
+         * If the Ignore receiver list is empty, prompt error message and skip the test.
+         */
+        if (ignoreReceiversNameList.isEmpty()) {
+            return bad("----> " + sftpcheckMessage + "FAILED: Ignore receiver list is empty.")
+        }
+
+        /**
+         * Start check the connection for each organization
+         */
+        ignoreReceiversNameList.forEach { receiver ->
+            /**
+             * Obtain the URL/path endpoint per environment (localhost or staging)
+             */
+            val path = environment.formUrl(sftpcheckUri + receiver).toString()
+
+            /**
+             * Check the organization ignore receiver connections
+             */
+            echo("SFTPCHECK Organizatin: $receiver...")
+
+            val (_, response, result) = sftpReceiverIgnoreOrganizationCheck(path, accessToken)
+            val (_, error) = result
+            if (response.statusCode == HttpStatus.SC_OK) {
+                good(
+                    "----> " + sftpcheckMessage + "PASSED with response code: " +
+                        " ${response.statusCode} "
+                )
+            } else {
+                sftpcheckTestResult = bad(
+                    "----> " + sftpcheckMessage + "FAILED with error code: : " +
+                        "${response.statusCode} - ${error?.response?.responseMessage}..."
+                )
+            }
+        }
+
+        return when (sftpcheckTestResult) {
+            true -> good("Test passed: SFTPCHECK ")
+            false -> bad(sftpcheckMessage + "Failed")
+        }
+    }
+
+    /**
+     * listReceiverNames - it extracts all receiver ignore organization from the database setting table
+     *  using setting API.
+     *
+     * @returns:
```

Comment:
Change to @return, there is no @returns

Reply:
If I remember correctly, the kdoc needs to be on the same line, like
@return I return this